conf/application.conf

```diff
@@ -149,8 +149,8 @@ processingBatchSize=4
 fileTypesFilter.allowedMimeTypes = [
   { user-agent: "business-rates-attachments", mime-types: "application/pdf,image/jpeg" },
   { user-agent: "customs-declarations", mime-types: "application/pdf,image/jpeg" },
-  { user-agent: "upscan-acceptance-tests", mime-types: "application/pdf" },
-  { user-agent: "Mozilla/4.0", mime-types: "application/pdf" }
+  { user-agent: "upscan-acceptance-tests", mime-types: "application/pdf,application/octet-stream" },
+  { user-agent: "Mozilla/4.0", mime-types: "application/pdf,application/octet-stream" }
```

Comment:
Hi Ash! Under what circumstances do we want to accept binary files for the acceptance tests and performance tests? I think releasing that will break the acceptance, as we check that binary files are not allowed.